src/shouldFail.js

```diff
@@ -29,7 +30,22 @@ async function shouldFail (promise) {
   await shouldFailWithMessage(promise);
 }
 
+async function withMessage (promise, message) {
+  // Find out if current version of ganache-core supports revert reason i.e >= 2.2.0.
```

Comment:
Can we add a link to the 2.2.0 release notes? https://github.com/trufflesuite/ganache-core/releases/tag/v2.2.0